src/admin.ts

```diff
@@ -20,4 +20,34 @@ export class Admin {
     );
     return response.data.token as string;
   }
+  async getAvailableFunctions({ customerId, namespace }: Runnable) {
+    const response = await axios.get(
+      `${this.baseUrl}/functions/${customerId}/${namespace}`
+    );
+    return response.data;
+  }
+  async getFunctionResults({
```

Comment:
Let's add some spacing between the functions

Reply:
@MacklinEngineering you can just run `npm run format` (and I'll set up CI to enforce it)

Reply:
handled in below commit